frontend/src/pages/register.tsx

```diff
@@ -86,7 +86,7 @@ export const RegisterPage: NextPage = () => {
             <Card className='bg-dark mb-4 text-start'>
               <Card.Body>
                 <Form onSubmit={doRegisterSubmit} className={'d-flex flex-column gap-3'}>
-                  <UsernameField onChange={onUsernameChange} value={username} />
+                  <UsernameLabelField onChange={onUsernameChange} value={username} isValid={Boolean(username.trim())} />
```

Comment:
`Boolean(username.trim())` is not a statically evaluated expression.
To improve performance, it should therefore be extracted and wrapped in a `useMemo`.

Reply:
sure @ErikMichelson, done